src/Raven.Client/Documents/Subscriptions/CreateSubscriptionResult.cs

```diff
@@ -3,6 +3,7 @@ namespace Raven.Client.Documents.Subscriptions
     public class CreateSubscriptionResult
     {
         public string Name { get; set; }
+        public long RaftIndex { get; set; }
```

Comment:
RaftCommandIndex to make it consistent with other places (please check if I'm correct)

Reply:
done